pages.zh/common/alacritty.md

```diff
@@ -0,0 +1,24 @@
+# alacritty
+
+> 跨平台，GPU 加速的终端模拟器。
```

Comment:
```suggestion
> 跨平台，GPU 加速的终端模拟器.
```

Should end with a period.

Reply:
This is the Chinese way to write a period, but right, those should be same in the whole page.